mkdocs.yml

```diff
@@ -61,9 +61,8 @@ nav:
   - Guides:
     - User Guides:
       - Getting started: guides/index.md
-      - Adapter Rollout: guides/adapter-rollout.md
```

Comment:
nit: I think keeping adapter rollout and pool rollout separate files is better. Here we can organize the navigation like 

```
-  Rollout
  - Adapter Rollout: guides/adapter-rollout.md
  - InferencePool Rollout: guides/inference-pool-rollout.md
 ```

Reply:
I eventually make three files, `rollout.md` will link to `adapter-rollout.md` and `nference-pool-rollout.md`